Source/AwsCommonRuntimeKit/crt/CommonRuntimeError.swift

```diff
@@ -1,6 +1,27 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0.
 
+import AwsCCommon
+
 public enum CommonRunTimeError: Error {
     case crtError(CRTError)
 }
+
+public struct CRTError {
+
```

Comment:
CRTError no longer conforms to Error protocol. We should never throw CRTError directly and it just complicates catching the error.